chapters/arrays.tex

```diff
@@ -969,15 +969,21 @@ \section{Array Indexing}\label{array-indexing}
 \end{nonnormative}
 
 \begin{example}
-% henrikt-ma: This listing needs to be cleaned up from confusing punctuation.
-\begin{lstlisting}[language=modelica, escapechar=!]
-a[:, j] !\emph{is a vector of the j-th column of a,}!
-a[j] !\emph{is a vector of the j-th row of a:}! a[j, :]
-a[j : k] is {[a[j], a[j+1], ... , a[k]}
-a[:,j : k] is [a[:,j], a[:,j+1], ... , a[:,k]],
-v[2:2:8] = v[ {2,4,6,8} ] .
-v[{j,k}] := {2,3}; // Same as v[j] := 2; v[k] := 3;
-v[{1,1}] := {2,3}; // Same as v[1] := 3;
+Array indexing expressions:
+\begin{lstlisting}[language=modelica]
+a[:, j]      // Vector of the j-th column of a.
+a[j]         // Vector of the j-th row of a. Same as: a[j, :]
+a[j : k]     // Same as: {[a[j], a[j+1], $\ldots$, a[k]}
```

Comment:
Should be: `Same as: {a[j], a[j+1], $\ldots$, a[k]}`

Reply:
Good catch!  I'm glad to se it wasn't introduced here.  Fixed.  Resolving.